base/commands/shell.go

```diff
@@ -103,6 +105,33 @@ func (cm *ShellCommand) ExecInteractive(ctx context.Context, ec plug.ExecContext
 	return sh.Start(ctx)
 }
 
+func prompt(cfgPath string) string {
+	if cfgPath == "" {
+		return "CLC> "
+	}
+	// Best effort for absolute path
+	if !filepath.IsAbs(cfgPath) {
+		p, err := filepath.Abs(cfgPath)
+		if err == nil {
+			cfgPath = p
+		}
+	}
+	pd := parentDir(cfgPath)
+	return fmt.Sprintf("CLC (%s) > ", dotify(pd, 12))
+}
+
+func parentDir(path string) string {
+	dirs := filepath.Dir(path)
+	return filepath.Base(dirs)
+}
```

Comment:
Could you move this function to `paths.ParentDir` ?

Reply:
Sure 👍🏼